deltachat-ios/Coordinator/AppCoordinator.swift

```diff
@@ -321,8 +325,20 @@ class SettingsCoordinator: Coordinator {
         let helpViewController = HelpViewController()
         navigationController.pushViewController(helpViewController, animated: true)
     }
+
+    func showDebugToolkit() {
+        DBDebugToolkit.setup(with: [])  // emtpy array will override default device shake trigger
+        DBDebugToolkit.setupCrashReporting()
```

Comment:
This means that crash reporting will only work if the user has opened the DebugToolkit once.  Maybe for reproducible bugs it's ok though...